src/SprykerEco/Zed/AvalaraTax/Business/Validator/CheckoutDataAddressValidator.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTax\Business\Validator;
+
+use Generated\Shared\Transfer\CheckoutDataTransfer;
+use Generated\Shared\Transfer\CheckoutErrorTransfer;
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use SprykerEco\Zed\AvalaraTax\Business\Executor\AvalaraResolveAddressExecutorInterface;
+
+class CheckoutDataAddressValidator implements CheckoutDataAddressValidatorInterface
+{
+    /**
+     * @var \SprykerEco\Zed\AvalaraTax\Business\Executor\AvalaraResolveAddressExecutorInterface
+     */
+    protected $avalaraResolveAddressExecutor;
+
+    /**
+     * @param \SprykerEco\Zed\AvalaraTax\Business\Executor\AvalaraResolveAddressExecutorInterface $avalaraResolveAddressExecutor
+     */
+    public function __construct(AvalaraResolveAddressExecutorInterface $avalaraResolveAddressExecutor)
+    {
+        $this->avalaraResolveAddressExecutor = $avalaraResolveAddressExecutor;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\CheckoutDataTransfer $checkoutDataTransfer
+     *
+     * @return \Generated\Shared\Transfer\CheckoutResponseTransfer
+     */
+    public function validateCheckoutDataShippingAddress(CheckoutDataTransfer $checkoutDataTransfer): CheckoutResponseTransfer
```

Comment:
* You have covered shipping to one address, but we also have to cover shipping to multiple addresses.
* Check `CheckoutDataTransfer::getShipments[]::getShippingAddress`.

Reply:
Resolved.